package.json

```diff
@@ -0,0 +1,927 @@
+    "unique-slug": "^2.0.2",
+    "unquote": "^1.1.1",
+    "unset-value": "^1.0.0",
+    "upath": "^1.2.0",
+    "unique-string": "^2.0.0",
+    "update-notifier": "^4.1.3",
+    "uri-js": "^4.4.1",
+    "url-loader": "^1.1.2",
+    "universalify": "^0.1.2",
+    "urix": "^0.1.0",
+    "url-parse-lax": "^3.0.0",
+    "url-parse": "^1.5.10",
+    "util": "^0.11.1",
+    "url": "^0.11.0",
+    "upper-case": "^1.1.3",
+    "util.promisify": "^1.0.0",
+    "util-deprecate": "^1.0.2",
+    "v8-compile-cache": "^2.3.0",
+    "utils-merge": "^1.0.1",
+    "utila": "^0.4.0",
+    "vary": "^1.1.2",
+    "uuid": "^3.4.0",
+    "use": "^3.1.1",
+    "uc.micro": "^1.0.6",
+    "validate-npm-package-license": "^3.0.4",
+    "vendors": "^1.0.4",
+    "verror": "^1.10.0",
+    "vue-hot-reload-api": "^2.3.4",
+    "vue-eslint-parser": "^9.1.0",
+    "vm-browserify": "^1.1.2",
+    "vue-template-es2015-compiler": "^1.9.1",
+    "vue-server-renderer": "^2.6.14",
+    "vue-style-loader": "^4.1.3",
+    "vue-loader": "^15.9.8",
+    "vuepress": "^1.9.9",
+    "vue": "^2.6.14",
+    "vuepress-html-webpack-plugin": "^3.2.0",
+    "vue-router": "^3.5.2",
+    "vue-template-compiler": "^2.6.14",
+    "vuepress-plugin-check-md": "^0.0.2",
+    "vuepress-plugin-container": "^2.1.5",
+    "vuepress-plugin-seo": "^0.1.4",
+    "vuepress-plugin-smooth-scroll": "^0.0.3",
+    "vuepress-plugin-zooming": "^1.1.8",
+    "vuepress-plugin-img-lazy": "^1.0.4",
+    "vuepress-plugin-sitemap": "^2.3.1",
+    "vuepress-theme-default-prefers-color-scheme": "^2.0.0",
+    "vuepress-plugin-meilisearch": "^0.13.0",
+    "vuex": "^3.6.2",
+    "watchpack": "^1.7.5",
+    "watchpack-chokidar2": "^2.0.1",
+    "webidl-conversions": "^4.0.2",
+    "webpack": "^4.46.0",
+    "vuepress-plugin-code-copy": "^1.0.6",
+    "webpack-dev-middleware": "^3.7.3",
+    "webpack-dev-server": "^3.11.2",
+    "webpack-log": "^2.0.0",
+    "webpack-sources": "^1.4.3",
+    "webpackbar": "^3.2.0",
+    "webpack-chain": "^6.5.1",
+    "whatwg-url": "^7.1.0",
+    "websocket-driver": "^0.7.4",
+    "when": "^3.6.4",
+    "webpack-merge": "^4.2.2",
+    "which-boxed-primitive": "^1.0.2",
+    "websocket-extensions": "^0.1.4",
+    "which-module": "^2.0.0",
+    "which": "^1.3.1",
+    "widest-line": "^3.1.0",
+    "wbuf": "^1.7.3",
+    "word-wrap": "^1.2.3",
+    "worker-farm": "^1.7.0",
+    "wrap-ansi": "^5.1.0",
+    "xdg-basedir": "^4.0.0",
+    "ws": "^6.2.2",
+    "xtend": "^4.0.2",
+    "write-file-atomic": "^3.0.3",
+    "y18n": "^4.0.3",
+    "yargs": "^13.3.2",
+    "wrappy": "^1.0.2",
+    "xmlbuilder": "^13.0.2",
+    "yargs-parser": "^13.1.2",
+    "zepto": "^1.2.0",
+    "yallist": "^3.1.1",
+    "zooming": "^2.1.1"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/meilisearch/documentation.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Is our license ISC? I thought it was MIT? Please check the LICENSE file and update this.

Reply:
It's MIT